tyr/servers/server.py

```diff
@@ -371,21 +386,58 @@ def blockdevicemapping(self):
 
         return bdm
 
+    def get_subnet_vpc_id(self, subnet_id):
+        vpcConn = VPCConnection()
+        subnets = vpcConn.get_all_subnets(
+            filters={'subnet_id': subnet_id})
+        if len(subnets) == 1:
+            vpc_id = subnets[0].vpc_id
+            return vpc_id
+        else:
+            self.log.error("None or greater then 1 subnet returned")
```

Comment:
If something happens which would cause the programme to exit, we should be using the `critical` log level.